header-footer-grid/Core/Magic_Tags.php

```diff
@@ -290,6 +290,16 @@ public function site_tagline() {
 		return get_bloginfo( 'description' );
 	}
 
+	/**
+	 * Title for the page currently being viewed.
+	 *
+	 * @return string
+	 */
+	public function current_query_title() {
+		$show_homepage_name = apply_filters( 'neve_query_title_magic_tag_on_homepage', false );
```

Comment:
@UVLabs this function will never run unless you add a filter in child-theme to change this value. It should work by default, no need to add any new filter.